ui/src/app/views/cohort-list/component.spec.ts

```diff
@@ -15,6 +15,7 @@ import {
   updateAndTick
 } from 'testing/test-helpers';
 
+import {CohortEditModalComponent} from '../cohort-edit-modal/component';
```

Comment:
not sure - so guess this is more a question than comment... But do we want relative paths? Or absolute ones? I've always preferred absolute ones, but figure we should be consistent.

Reply:
Agreed. Absolute ones seem better. Will swap!